go/vt/sqlparser/sql.y

```diff
@@ -5171,6 +5215,205 @@ trim_type:
     $$ = TrailingTrimType
   }
 
+frame_units:
+  ROWS
+  {
+    $$ = FrameRowsType
+  }
+| RANGE
+  {
+    $$ = FrameRangeType
+  }
+
+frame_point_type:
+  CURRENT ROW
+  {
+    $$ = CurrentRowType
+  }
+| UNBOUNDED PRECEDING
+  {
+    $$ = UnboundedPrecedingType
+  }
+| UNBOUNDED FOLLOWING
+  {
+    $$ = UnboundedFollowingType
+  }
```

Comment:
This rule is used only in 1 place. We should unroll this definition, since this rule does not actually list all the frame point types (`ExprPrecedingType` is not here) and it might someone else later to use this rule incorrectly